shell/assets/translations/zh-hans.yaml

```diff
@@ -4664,6 +4740,7 @@ tableHeaders:
   id: ID
   image: 镜像
   imageSize: 大小
+  ingressClass: Ingress 类
```

Comment:
`IngressClass` should be fine.
refer to: https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-class